tests/robot-tests/README.md

```diff
@@ -27,17 +29,43 @@
 - [**Who should I talk to?**](#who-should-i-talk-to)
 
 
-## **What is this?**
+## What is this?
 This test framework runs UI tests against the Explore Education Statistics service using Selenium and Robot Framework.
 
-Currently, these tests are being maintained so they can be run on Windows. They're not being maintained for Linux or MacOS, but they may work without too much trouble. 
+Currently, these tests are being maintained so they can be run on Windows and Ubuntu
 
-## **What do I need to install?**
 
-Firstly, install Python 3.10 or greater
-   * For Windows, you'll need to download Python3 from here: https://www.python.org/downloads/
-   * For Linux, use the package manager (i.e. On Ubuntu, `sudo apt-get install python3.10`)
-   * For Mac, use the package manager: `brew install python@3.10`. You then need to change the default python symlink to the version you've just installed `ln -s -f /usr/local/bin/python3.10 /usr/local/bin/python` 
+## Pyenv installation
+  Pyenv is a tool for installing and managing multiple versions of Python on a single machine. It is the recommended way of installing and managing python.
+  
+  Install pyenv using [chocolatey](https://chocolatey.org/install)
+
+  ```bash
+  choco install pyenv-win
+  pyenv install 3.101
+  ```
+
+  **Linux**
+
+  ```bash
+  curl https://pyenv.run | bash
+  pyenv install 3.101
+
+  ```
+
+  **Mac**
+
+  ```bash
+  brew install pyenv
+  pyenv install 3.101
+  ```
```

Comment:
I'd still be inclined to replace all this with a link to the pyenv github installation instructions (https://github.com/pyenv/pyenv#installation), and then just say you need to run `pyenv install 3.10`.